pandas/tests/test_generic.py

```diff
@@ -415,6 +415,10 @@ def test_sample(self):
                 o.sample(frac=0.7, random_state=np.random.RandomState(test)),
                 o.sample(frac=0.7, random_state=np.random.RandomState(test)))
 
+            np.random.seed(test)
+            self._compare(o.sample(n=4), o.sample(n=4))
```

Comment:
You're right. I fixed the test.

On 05/12, Stephan Hoyer wrote:
> > @@ -415,6 +415,10 @@ def test_sample(self):
> >                  o.sample(frac=0.7, random_state=np.random.RandomState(test)),
> >                  o.sample(frac=0.7, random_state=np.random.RandomState(test)))
> >  
> > +            np.random.seed(test)
> > +            self._compare(o.sample(n=4), o.sample(n=4))
> 
> This test doesn't look right to me. You need to call `np.random.seed(test)` before every call to `sample`, not just the first one.
> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/pydata/pandas/pull/13161/files/08b2cd1ae1e581e720d169cbf8d4888adcb07e67#r63099499